docs_template/source/getting_started/template_setup.md

```diff
@@ -83,8 +70,6 @@ placeholder names, email addresses, and repository links with your own:
 
 - `LICENSE`
 
```

Comment:
Just include a line telling them to delete the `.github` folder here. Incorporate #171, so that #161 is closed here.